app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -47,6 +47,23 @@ public TodoTxtTextActions(Activity activity, Document document) {
 
     @Override
     public boolean runAction(String action, boolean modLongClick, String anotherArg) {
+        if (action == CommonTextActions.ACTION_SEARCH ) {
+            final String origText = _hlEditor.getText().toString();
+
+            SearchOrCustomTextDialogCreator.showSearchDialog(_activity, origText,
```

Comment:
Should test with `instanceof` and consider adding a settings switch if performance is a concern.